controllers/Redirects.php

```diff
@@ -138,6 +138,30 @@ public function update($recordId = null, $context = null)
         parent::update($recordId, $context);
     }
 
+    /**
+     * Force rule flushing on update
+     */
+    public function update_onSave(?string $context = null)
+    {
+        /** @noinspection PhpPossiblePolymorphicInvocationInspection */
```

Comment:
```suggestion
```
We can just remove this line. @bennothommo he copied from elsewhere in the file (the create_onSave()), but the asExtension way of achieving the same result is more explicit and doesn't require this comment. I'm assuming VDLP had some form of static analysis in their editors or in a git hook or something that yelled at them when they did this and they added the comment to shut it up instead of just fixing the problem to use the asExtension() method.

Reply:
Probably some PHPStorm bullshit - I know that encouraged them to add a lot of this cruft.

Reply:
Removed in 21326ac38c8e02d76d9e1a8845d8e52351853623